docs/dictionary/command/send.lcdoc

```diff
@@ -38,21 +38,21 @@ message: An expression that evaluates to a message name, possibly including para
+The result: The ID of the <message> in the <pendingMessages> <function> queue.
 
 Description:
-Use the <send> <command> to override the normal <message path>, or to delay a <command> until a specified time. 
+Use the <send> <command> to override the normal <message path>, or to delay a <command> until after a specified time.
 
-The <message> can be either the name of a <handler> or a <LiveCode> <statement>. If the <message> is a <literal string> containing more than one <word> (for example, if it includes <parameter|parameters>, or if it is a multi-word command), it must be enclosed in <double quote|double quotes>. 
+The <message> can be either the name of a handler or a <LiveCode> <statement>. If the <message> is a <literal string> containing more than one <word> (for example, if it includes <parameter|parameters>, or if it is a multi-word command), it must be enclosed in <double quote|double quotes>.
 
 Any parameters are evaluated before they are passed to the <send> <command>. For example, the following statement sends the <mouseUp> <message> with 2 as the first parameter:
 
   send "mouseUp 1+1" to button "Example"
 
-When the send command is used the stack containing the target handler temporarily becomes the defaultStack. All object references in the <message> are <evaluate|evaluated> in the current context i. e. the defaultStack. Therefore references within the message that refer to "this card" or "this stack" will be referring to the card or stack where the target handler is located. 
+>*Important:* When the send command is used, the stack containing the target handler temporarily becomes the defaultStack. All object references in the <message> are <evaluate|evaluated> in the current context i. e. the defaultStack. Therefore references within the message that refer to "this card" or "this stack" will be referring to the card or stack where the target handler is located.
 
->*Important:*  Specifying a <time> can affect the order in which <statement|statements> are <execute|executed>. If you don't specify a <time>, the <message> is sent immediately, and any <handler> it <trigger|triggers> is completed before the rest of the current <handler> is <execute|executed>. If you use the send in time form of the <send> <command> --even if you specify a <time> of zero seconds--the current handler finishes <execute|executing> before the <message> is sent.
+>*Important:*  If you don't specify a <time>, the <message> is sent immediately, pausing and interrupting any currently executing handler.  Any handler it <trigger|triggers> is completed before the rest of the current handler is <execute|executed>. However, if you specify a <time>, even one of 0, the current handler finishes <execute|executing> before the <message> is sent.
```

Comment:
I don't think
```
, pausing and interrupting any currently executing handler
```
is necessary, as the meaning is clarified in the following clause:
```
Any handler it <trigger|triggers> is completed before the rest of the current handler is <execute|executed>
```